samples/snippets/quickstart.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigqueryconnection_quickstart]
+import argparse
+
+from google.cloud import bigquery_connection_v1 as bq_connection
+
+
+def main(project_id: str = "your-project-id", location: str = "US") -> None:
+    # Constructs client for interacting with the service
+    client = bq_connection.ConnectionServiceClient()
```

Comment:
Can we push the client initialization down into `list_connections`?

Reply:
Addressed in latest commit! I also removed the client fixture from `conftest.py` though I'm guessing we'll need it with future samples!